dataset/dictionary/locations.json5

```
@@ -247,111 +283,130 @@
   {
     en: "Feiyun Slope",
     ja: "緋雲の丘",
+    zhCN: "绯云坡",
     pronunciationJa: "ひうんのおか",
     tags: [ "liyue", "location" ],
   },
   {
     en: "Yujing Terrace",
     ja: "玉京台",
+    zhCN: "玉京台",
     pronunciationJa: "ぎょくけいだい",
     tags: [ "liyue", "location" ],
   },
   {
     en: "Yuehai Pavilion",
     ja: "月海亭",
+    zhCN: "月海亭",
     pronunciationJa: "げつかいてい",
     tags: [ "liyue", "location" ],
   },
   {
     en: "Chihu Rock",
     ja: "チ虎岩",
+    zhCN: "吃虎岩 / 螭虎岩",
+    notes: "中国語の「螭虎岩」は古い時代の表記で、現在は簡略化されて「吃虎岩」と表記されるようになったという設定。 / 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。"
```

Oops, I forgot the trailing comma.

```suggestion
    notes: "中国語の「螭虎岩」は古い時代の表記で、現在は簡略化されて「吃虎岩」と表記されるようになったという設定。 / 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。",
```